tern/report/report.py

```diff
@@ -143,16 +145,23 @@ def generate_format(images, format_string):
             name=format_string,
             invoke_on_load=True,
         )
-        return mgr.driver.generate(images)
+        if(format_string == 'html'):
```

Comment:
Don't need these changes here. `mgr.generate` will use the format input from the user to select the correct formatting style.